docs/HowToRunSTDKSample.md

```diff
@@ -0,0 +1,258 @@
+# How to develop things using STDK based TizenRT   
+STDK (Smart Things Develop Kit) is one of the features of the Smart Things ecosystem. This guide describes with an example, how to enable STDK in TizenRT and develop applications using it.
+
+We choose a common example of a blinking Led to demonstrate how to develop a so-called LED Things and control it via a SmartThings mobile app.
+
+## Contents
+* [Prerequisites](#prerequisites)
+* [Get TizenRT Source](#get-tizenrt-source)
+* [Create device identity](#create-device-identity)
+* [Register Device Profile on SmartThings Devworkspace](#register-device-profile-on-smartthings-devworkspace)
+* [Build and Flash](#build-and-flash)
+* [Try to do EasySetup and Control](#try-to-do-easysetup-and-control)
+* [How to develop your STDK device](#how-to-develop-your-stdk-device)
+
+## Prerequisites   
+This section will describe the pre-requisite steps to install and build STDK for the ESP-WROVER-KIT board.   
+
+* Board specific prerequisites [[Details]](../README.md#supported-board--emulator).   
+* Toolchain installation [[Details]](https://smartthings.developer.samsung.com/docs/devices/direct-connected-devices/setup-environment.html).   
+
+### Get TizenRT Source   
+#### Clone TizenRT source code.   
+```bash
+git clone https://github.com/Samsung/TizenRT.git
+cd TizenRT
+TIZENRT_BASEDIR="$PWD"
+```
+#### Apply patch for STDK.   
++ Update submodule   
+```bash
+git submodule update --init "external/stdk/st-device-sdk-c"
+git submodule update --init "external/libsodium/libsodium"
+```
++ Apply patch for stdk   
+```bash
+cd external/stdk/st-device-sdk-c
+git apply ../patch/0001-port-net-fix-ssl-issue-on-TizenRT.patch
+```
```

Comment:
applying a patch is not needed anymore so please delete it

Reply:
Removed Those lines.